formula-coroutines/src/main/java/com/instacart/formula/coroutines/FlowAction.kt

```diff
@@ -24,20 +27,10 @@ interface FlowAction<Event> : Action<Event> {
          * }
          * ```
          */
-        inline fun <Event> fromFlow(
-            scope: CoroutineScope = MainScope(),
-            crossinline create: () -> Flow<Event>
+        fun <Event> fromFlow(
+            create: () -> Flow<Event>
         ): Action<Event> {
-            return object : FlowAction<Event> {
-
-                override val scope: CoroutineScope = scope
-
-                override fun flow(): Flow<Event> {
-                    return create()
-                }
-
-                override fun key(): Any = Unit
-            }
+            return FlowActionImpl(null, create)
```

Comment:
won't sending a null key have clashes with others who call with a null key?

Reply:
Maybe I am remembering wrong on what this key is for

Reply:
No, because composite keys are used 
```kotlin
Key(
    firstPart = transition::class,
    secondPart = action.key()
)
```